packages/schema/lib/schemas/AuthFieldSchema.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const makeSchema = require('../utils/makeSchema');
+const FieldSchema = require('./FieldSchema');
+
+module.exports = makeSchema(
+  {
+    ...FieldSchema.schema,
+    id: '/AuthFieldSchema',
+    description: 'Field schema specialized for authentication fields.',
+    properties: {
+      ...FieldSchema.schema.properties,
+      isSafe: {
+        description:
+          'Indicates if this authentication field is safe (not secret).',
+        type: 'boolean',
+      },
+    },
+
+    // Do not allow certain isSafe = True if sensitive tokens exists
+    not: {
+      properties: {
+        key: {
+          enum: [
+            'access_token',
+            'refresh_token',
+            'api_key',
+            'password',
+            'secret',
```

Comment:
@FokkeZB In case you have more auth fields to add to the list of fields that cannot be marked as isSafe.

Reply:
@kola-er you may want to copy https://gitlab.com/zapier/team-developer-platform/secret-scrubber-js/-/blob/main/src/convenience.ts?ref_type=heads#L6-L19